include/seqan/platform.h

```diff
@@ -393,7 +409,11 @@ typedef int8_t __int8;     // nolint
     #ifdef STDLIB_VS
     #define SEQAN_HAS_EXECINFO 0
     #elif defined(__has_include)
-    #define SEQAN_HAS_EXECINFO !!__has_include(<execinfo.h>)
+        #if __has_include(<execinfo.h>)
+        #define SEQAN_HAS_EXECINFO 1
+        #else
+        #define SEQAN_HAS_EXECINFO 0
+        #endif
     #else // assume that it is there
```

Comment:
We also reverted the expression down below, because it caused problems. I can't remember the expression, but something like `!SEQAN_ASYNC_IO && OTHER_CONSTANT` resulted in a compiler error.

```c++
     #elif defined(__has_include) && defined(__unix__)
         #if __has_include(<aio.h>)
         #define SEQAN_ASYNC_IO 1
         #else
         #define SEQAN_ASYNC_IO 0
         #endif
#endif
```